charts/pega/values-large.yaml

```diff
@@ -148,6 +148,8 @@ global:
         # Inactivity time after which requestor is passivated
         passivationTimeSec: 900
 
+      # Port 7003 turned ON by default in all installs except large.
+      # comment following lines NOT to expose service and ports.
```

Comment:
Port 7003 is exposed by default in all installs except large deployments.
 To NOT to expose service and ports in your deployment, you must comment out the following lines.